Little changes

KingnotPy opened the pull request:
Hello, 
I changed the following line to prevent selection of all the "table" elements in the region

KingnotPy committed:
Update README.md

KingnotPy committed:
Update fapex.js line 120 

changed line 120 so it only targets the table element that contains the data

mrfarhanakram commented:
Thanks, KingnotPy! This change is working perfectly for classic reports but it's not working for Interactive Reports. I'll be happy to merge it if this issue is fixed.

KingnotPy committed:
Update fapex.js - Fix on IRRs

Fixed table element targeting on Interactive Reports as well.
Fixed clicking on Interactive Report headers breaking sessions

KingnotPy commented:
Fixed the issue,
And also found and fixed clicking on headers on interactive reports breaking sessions

mrfarhanakram commented:
Thanks for your contribution :)
